CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing to Topo
+👋🏻 Welcome to Topo by CabinDAO! It's great you're here.
+
+## Onboarding
+1. Join Cabin by joining our [Discord](https://discord.com/invite/ttgRU7QKVE) and following the path laid out in the [#01-welcome-to-cabin](https://discord.com/channels/849304522500210709/936441783808851998) channel.
+
+2. After you onboard onto Cabin, you can ask for access to our [bounty board](https://app.clarity.so/cabin/view/9849fa0e-9cd6-4805-bcce-5928bd633b07) that shows tasks we need help with. 
+
+3. You then need to onboard into the Product Guild by following the instructions in the Product Guild [#start-here](https://discord.com/channels/849304522500210709/936372031874818168) channel.
+
+4. Now you're ready to request bounties! Check out [#110 Write SETUP.md for Topo](https://app.clarity.so/cabin/view/9849fa0e-9cd6-4805-bcce-5928bd633b07) as a possible task to take on. 

```

Comment:
This line will become stale once this task is completed right? This is also the wrong link for the task anyway, task is here: https://app.clarity.so/cabin/work/110

Reply:
I am going from the [spec](https://app.clarity.so/cabin/work/111) for the bounty and interpreted "Call to action to review the #110 Write SETUP.md for Topo (5 ₡ABIN)" as this. I am fixing the link, but please let me know if you want me to remove it altogether.

cc @ZakkMan who wrote the spec

Reply:
I think it's meant to review the SETUP.md doc directly, as in "follow the instructions here to get setup to then contribute" rather than the clarity task specifically

Reply:
Hey y'all, yea - we want to call out for them to follow the instructions in SETUP.md to get the project set up. Though @dvargas92495, correct me if I'm wrong, did we decide to merge SETUP.md with the README.md?

Reply:
That was in Passports, in general I also support merging it in most repos. 

But as it stands today, SETUP exists, so I think it's find to link for now to just get the info live, whoever wants to merge could relink appropriately